.github/actions/hugo-build-action/action.yaml

```diff
@@ -0,0 +1,23 @@
+name: Hugo Build
+description: Builds documentation using Hugo
+inputs:
+  BASE_URL:
+    required: false
+    description: Optional base url used during build
+
+runs:
+  using: "composite"
+  steps:
+    - name: Setup Hugo
+      uses: peaceiris/actions-hugo@v2
+      with:
+        hugo-version: '0.81.0'
+        extended: true
+    - name: Install dependencies
+      working-directory: gooddata-sdk/docs/hugo/docs
+      run: npm ci
+      shell: bash
+    - name: Build
```

Comment:
Should not we cover build and deps install by make command? It will make it easy to trigger build on localhost.

Reply:
Will be resolved in follow-up.